website/docs/developer/development-guide/code/best-practices.mdx

```diff
@@ -0,0 +1,72 @@
+---
+id: best-practices
+title: Best Practices
+description: Best practices for coding in ORCA.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Unit Testing
+All code should reach minimum 80% coverage through [Unit Tests](unit-tests.md).
+
+## Code Style
+We use the [Google Style Guide](https://google.github.io/styleguide/pyguide.html) for style elements such as documentation, titling, and structure.
+
+## Stop on Failure
+Failures within ORCA break through to the Cumulus workflow they are a part of.
+To this end, raising an error is preferred over catching the error and returning a null value or error message.
+
+```python
+try:
+    value = function(param)
+except requests_db.DatabaseError as err:
+    logging.error(err)
+    raise
+```
+```python
+if not success:
+    logging.error(f"You may log additional information if desired. "
+                  f"param: {param}")
+    raise DescriptiveErrorType(f'Error message to be raised info Cumulus workflow.'
+```
+Retries can then be configured in the workflow json if desired. See
```

Comment:
Maybe put a comment in here about this is the "preferred" way to manage retries instead of in code?

Reply:
Feels like we covered that in the description above. Don't think it'd fit to call it out here, especially since this is just one of two examples on the preferred way.

Reply:
I should have been more clear. I guess I was thinking of the "if desired" in the sentence. But I think it is ok as is after pointing out the paragraph above.